src/common/components/Tooltip/Skin/index.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import React from 'react';
+import ItemHeader from '../ItemHeader';
+import Background from '../Background';
+import styles from './styles.less';
+
+type Props = {
+  data: Object,
+};
+
+const SkinTooltip = ({ data: { skin } }: Props) => {
+  return (
+    <Background>
+      <ItemHeader name={skin.name} icon={skin.icon} rarity={'white'} />
```

Comment:
rarity could just be `rarity="white"`

Reply:
Ah, did not know that. I am new to React.